packages/ng-schematics/package.json

```diff
@@ -3,16 +3,15 @@
   "version": "0.1.0",
   "description": "Puppeteer Angular schematics",
   "scripts": {
-    "dev": "npm run build --watch",
-    "dev:test": "npm run test --watch",
-    "copy": "wireit",
+    "build:copy": "wireit",
```

Comment:
I think we can Upgrade Wireit and remove these `build:*` steps as it now supports independent scripts in `wireit` config part.

Reply:
Let's do this in a follow up.